spacesuit/src/value.rs

```diff
@@ -165,9 +177,13 @@ impl Neg for SignedInteger {
     }
 }
 
+/// Extension trait for committing Values to the Prover's constraint system.
+/// TBD: make this private by refactoring the benchmarks.
 pub trait ProverCommittable {
+    /// Type for a result of committing this type to a constraint system.
```

Comment:
This is a confusing statement... is "this type" the `ProverCommittable` type? Maybe stating that explicitly would make this statement less weird

Reply:
how about this? 
https://github.com/interstellar/slingshot/pull/115/commits/6bb204b059dafc2f0cb4a2ab589c716f0711e69c

> Result of committing Self to a constraint system.

Reply:
👍